### ISSUE TITLE
Shell integration in windows 11 when using classic context menus doesn't appear to work

### ISSUE THREAD

**Issue by stickynoteme** [sandboxie-plus/Sandboxie#2625]
### Describe what you noticed and did

Using the classic context menu in windows 11, and enabling shell integration in options doesn't yield any new entries in the context menu.

P/s
This is my first time installing sandboxie Plus on windows 11, but I have been using it on windows 10 for years.

### Affected program

Windows shell

### Download link

www.windows.com

### Where is the program located?

The program is installed both inside and outside the sandbox.

### Expected behavior

Context menu entries

### What is your Windows edition and version?

Windows 11 Pro 22h2

### In which Windows account you have this problem?

A local or Microsoft account without special changes.

### Please mention any installed security software

Windows defender

### What version of Sandboxie are you running?

1.6.6 downloaded 2023-01-22

### Is it a new installation of Sandboxie?

I recently did a new clean installation.

### In which sandbox type you have this problem?

Not relevant to my request.

### Can you reproduce this problem on a new empty sandbox?

Not relevant to my request.

**Comment by DavidXanatos** [comment]
on my test vm i also use classic menus and for me it works fine, anyone else having this issue?

**Comment by stickynoteme** [comment]
After a reinstall it's working directly on .exe files now, but still not showing up on program shortcuts.

**Comment by DavidXanatos** [comment]
and if you reboot your PC does this help?
there is no good reason why this should work only on some file types :/

**Comment by stickynoteme** [comment]
A reboot didn't work, but that might have been because I installed / uninstalled / installed all on one boot and things got out or wack.
So I:
Uninstalled > Rebooted
Installed > Rebooted
Changed the shell integrations settings > Rebooted
And now things are working right. 

as an unrelated personal common, boot times nowadays are insane... there's reasons I've started avoiding restarting my computer. Takes almost 5mins per reboot on my brand new computer, compared to the whooping 7 seconds it took my not that ancient skylake system to reboot.

**Comment by jackuars7** [comment]
> as an unrelated personal common, boot times nowadays are insane... there's reasons I've started avoiding restarting my computer. Takes almost 5mins per reboot on my brand new computer, compared to the whooping 7 seconds it took my not that ancient skylake system to reboot.

This is true if you upgraded your Windows from 10 to 11. I nearly reached the point of insanity when it takes 5 minutes to reboot. After doing a clean install of Windows 11 (removing all program files/folders in C:\drive) and then applying all the Windows updates and also driver updates with IOBit Driver Booster (yes, it's safest out there), it runs like a Ferrari now even with all the old programs installed back. I strongly suggest you do this.

**Comment by chaosdrop** [comment]
> Reduced my boot time from 2 minutes to ~7-12 seconds overnight and applications runs noticeably much faster now.

Interesting, and thanks for the info. This computer has never had anything but windows 11 on it. it's an AMD 7900x / AM5 setup. 
Now I did accidentally knock out my USB hub the other day and that reduced boot time by almost a whole minute.  Because of that I did learn my external backup drives are part of the reasons for the super slowness. So I might look into a USB hub with a physical switch so I can turn off external drives during boot.

The IOBit Driver Booster is an interesting one I might look into, because even after the bios F2/del windows still takes a good minute+ to actually get to the login screen, and I've learned that ASRock is super slow with posting official drivers to their support pages.
